### DISCUSSION TITLE
Max age retirement of batteries

### DISCUSSION THREAD

**Discussion by ktehranchi** [NREL/ReEDS-2.0#184] [Q&A: Data] [answered]
Hi there-

I'm wondering why batteries' maximum age is constrained to 15 years, whereas the capital recovery period listed in the most recent ATBs is 20/30 years.

maxage referenced: https://github.com/NREL/ReEDS-2.0/blob/e65ed5ed4ffff973071839481309f77d12d802cd/inputs/plant_characteristics/maxage.csv 

Am I interpreting the maxage usage in the model correctly?

**Comment by wesleyjcole** [comment, marked as answer]
The capital recovery period in the ATB is only used to calculated the LCOE values.  There are no storage LCOE values for storage in the ATB, so that 20/30 capital recovery lifetime is not used.  The ATB specifies that batteries have a 15-year lifetime (https://atb.nrel.gov/electricity/2024/utility-scale_battery_storage), which is consistent with what we are modeling in ReEDS.  Lifetime values from literature tend to be in the 15-20 year range:

![image](https://github.com/user-attachments/assets/0f2b3c9a-02b9-4e14-91de-cfdd82da12a6)
 
(figure from Figure 6 of https://www.nrel.gov/docs/fy23osti/85332.pdf)
